tests/e2e/triggers/common.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+import typing
+
+import mgclient
+
+
+def execute_and_fetch_all(cursor: mgclient.Cursor, query: str, params: dict = {}) -> typing.List[tuple]:
+    cursor.execute(query, params)
+    return cursor.fetchall()
+
+
+def connect(autocommit=True, **kwargs) -> mgclient.Connection:
+    connection = mgclient.connect(host="localhost", port=7687, **kwargs)
+    connection.autocommit = autocommit
+    yield connection
```

Comment:
Sorry if I was unclear, but the idea of yield is to continue the fixture clean up, so basically what would you do now, after yield is:
```python
    triggers_list = execute_and_fetch("SHOW TRIGGERS;")
    for trigger in triggers_list:
      execute_and_fetch_all(cursor, f"DROP TRIGGER {trigger[name]}")
    execute_and_fetch_all(cursor, "MATCH (n) DETACH DELETE n")
```
and remove all cleanup logic from test cases.